scripts/install-freebsd-multi-user.sh

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+# System specific settings
+# FreeBSD typically uses UIDs from 1001+ for regular users,
+# so we'll use a range that's unlikely to conflict
+export NIX_FIRST_BUILD_UID="${NIX_FIRST_BUILD_UID:-30001}"
+export NIX_BUILD_GROUP_ID="${NIX_BUILD_GROUP_ID:-30000}"
+export NIX_BUILD_USER_NAME_TEMPLATE="nixbld%d"
+
+# FreeBSD service paths
+readonly SERVICE_DEST=/usr/local/etc/rc.d/nix-daemon
+
+poly_cure_artifacts() {
+    :
+}
+
+poly_service_installed_check() {
+    if [ -f "$SERVICE_DEST" ]; then
+        return 0
+    else
+        return 1
+    fi
+}
+
+poly_service_uninstall_directions() {
+    cat <<EOF
+$1. Delete the rc.d service
+
+  sudo service nix-daemon stop
+  sudo rm -f $SERVICE_DEST
+  sudo sysrc -x nix_daemon_enable
+
+EOF
+}
+
+poly_service_setup_note() {
+    cat <<EOF
+ - create an rc.d service (at $SERVICE_DEST) for nix-daemon
+
+EOF
+}
+
+poly_extra_try_me_commands() {
+    cat <<EOF
+  $ sudo service nix-daemon start
+EOF
+}
+
+poly_configure_nix_daemon_service() {
+    task "Setting up the nix-daemon rc.d service"
+
+    # Ensure the rc.d directory exists
+    _sudo "to create the rc.d directory" \
+          mkdir -p /usr/local/etc/rc.d
+
+    # Create rc.d script for nix-daemon
+    cat > "$SCRATCH/nix-daemon" <<'EOF'
+#!/bin/sh
+#
+# PROVIDE: nix_daemon
+# REQUIRE: DAEMON
+# KEYWORD: shutdown
+#
+# Add the following lines to /etc/rc.conf to enable nix-daemon:
+#
+# nix_daemon_enable="YES"
+#
+
+. /etc/rc.subr
+
+name="nix_daemon"
+rcvar="nix_daemon_enable"
+
+load_rc_config $name
+
+: ${nix_daemon_enable:=NO}
+
+command="/nix/var/nix/profiles/default/bin/nix-daemon"
+command_args=""
+pidfile="/var/run/nix-daemon.pid"
+
+start_cmd="${name}_start"
+stop_cmd="${name}_stop"
+
+nix_daemon_start() {
+    echo "Starting ${name}."
+    /usr/sbin/daemon -c -f -p ${pidfile} ${command} ${command_args}
+}
+
+nix_daemon_stop() {
+    if [ -f ${pidfile} ]; then
+        echo "Stopping ${name}."
+        kill -TERM $(cat ${pidfile})
+        rm -f ${pidfile}
+    else
+        echo "${name} is not running."
+    fi
+}
+
+run_rc_command "$1"
+EOF
```

Comment:
Should this be its own `*.in` file, with `@.....@` substitutions, and part of the build proper?

Reply:
Did that. I have to re-test.